src/main/java/com/exasol/adapter/jdbc/AbstractRemoteMetadataReader.java

```diff
@@ -91,22 +92,39 @@ public IdentifierConverter getIdentifierConverter() {
 
     @Override
     public SchemaMetadata readRemoteSchemaMetadata() {
+        return readRemoteSchemaMetadataForSelectedTables(List.of());
```

Comment:
Using an empty list to say "all" is counter-intuitive.

Reply:
Agreed to do it in a next PR as we need to change another interface